code/day2-operations/deployment/k8s/otel-agent-mtls.yaml

```diff
@@ -140,19 +151,21 @@ spec:
       labels:
         app: opentelemetry
         component: otel-agent
+        sidecar.istio.io/inject: "true"        
       annotations:
         proxy.istio.io/config: |
           # configure an env variable `OUTPUT_CERTS` to write certificates to the given folder
           proxyMetadata:
             OUTPUT_CERTS: /etc/istio-output-certs
-        sidecar.istio.io/inject: "true"
         sidecar.istio.io/userVolumeMount: '[{"name": "istio-certs", "mountPath": "/etc/istio-output-certs"}]'
         traffic.sidecar.istio.io/includeInboundPorts: ""
         traffic.sidecar.istio.io/includeOutboundIPRanges: ""
     spec:
       containers:
       - name: otel-agent
         image: otel/opentelemetry-collector-contrib:latest
+        args:
+          - --config=/etc/otel/config.yaml        
```

Comment:
new requirement of otel-agent image https://github.tools.sap/apm/kyma-extension/commit/4d5590e65ead3a916b09d8bee1e8e1902b0e6959